lvsfunc/util.py

```diff
@@ -622,18 +599,21 @@ def colored_clips(amount: int,
 
     Written by `Dave <mailto:orangechannel@pm.me>`_.
```

Comment:
Might want to put his github like for everyone else?
Mail is present there too.
https://github.com/OrangeChannel